### PR TITLE
Feature/40278

### PR DESCRIPTION
**Pull request by szafran89**

### CODE REVIEW

File: Snowdog_Menu/templates/main-mobile/menu.phtml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                <button
+                    class="side-menu__trigger"
+                    type="button"
+                    aria-label="<?= /* @escapeNotVerified */ __('Side menu close') ?>"
+                >
+                    <svg class="icon side-menu__trigger-icon">
+                        <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#close') ?>"></use>
+                    </svg>
+                    <span class="header-button__text">
+                        <?= __('Menu') ?>
```
Comment (Aekal) [inline Snowdog_Menu/templates/main-mobile/menu.phtml]: Please add `/* @escapeNotVerified */`

File: Snowdog_Menu/templates/main-mobile/menu.phtml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <button
+            class="side-menu__trigger"
+            type="button"
+            aria-label="<?= /* @escapeNotVerified */ __('Side menu open') ?>"
+        >
+            <svg class="side-menu__trigger-icon">
+                <use xlink:href="<?= $block->getViewFileUrl('/images/icons-sprite.svg#mobile-menu') ?>"></use>
+            </svg>
+            <span class="side-menu__trigger-text">
+                <?= __('Menu') ?>
```
Comment (Aekal) [inline Snowdog_Menu/templates/main-mobile/menu.phtml]: Please add `/* @escapeNotVerified */`

File: Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        $wrapperAttributes = [
+            'class' => [
+                $level > 1 ? 'list' : '',
+                $menuClass . '__inner-list',
+                $menuClass . '__inner-list--level' . $level
+            ],
+            'data-menu' => 'menu-' . $parentNode->getNodeId()
+        ];
+    ?>
+    <ul<?= $block->buildAttrFromArray($wrapperAttributes) ?>>
```
Comment (Aekal) [inline Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml]: There is no need for space after `<ul`?

File: Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                    $nodeAttributes['class'][] = $menuClass . '__inner-item--parent';
+                }
+
+                if ($node->getClasses()) {
+                    $nodeAttributes['class'][] = $node->getClasses();
+                }
+            ?>
+            <li<?= $block->buildAttrFromArray($nodeAttributes) ?>>
+                <?php echo $block->renderMenuNode($node) ?>
+                <?php echo $block->renderSubmenu($children, $node, $childrenLevel) ?>
```
Comment (Aekal) [inline Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml]: You can use shorthand

File: Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+                if ($children) {
+                    $nodeAttributes['class'][] = $menuClass . '__inner-item--parent';
+                }
+
+                if ($node->getClasses()) {
+                    $nodeAttributes['class'][] = $node->getClasses();
+                }
+            ?>
+            <li<?= $block->buildAttrFromArray($nodeAttributes) ?>>
```
Comment (Aekal) [inline Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml]: There is no need for space after <li?

File: Snowdog_Menu/templates/main-mobile/menu.phtml (hunk 1 above)
Comment (szafran89) [reply Snowdog_Menu/templates/main-mobile/menu.phtml]: added

File: Snowdog_Menu/templates/main-mobile/menu.phtml (hunk 2 above)
Comment (szafran89) [reply Snowdog_Menu/templates/main-mobile/menu.phtml]: added

File: Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml (hunk 3 above)
Comment (szafran89) [reply Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml]: yes there is no need

File: Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml (hunk 5 above)
Comment (szafran89) [reply Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml]: yes there is no need

File: Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml (hunk 4 above)
Comment (szafran89) [reply Snowdog_Menu/templates/main-desktop/menu/sub_menu.phtml]: changed

**Pull request merged**